benchmarks/text_hybrid_scoreboard/lexical_common.py

```
@@ -0,0 +1,507 @@
+#!/usr/bin/env python3
+"""Frozen corpus, reference semantics, validation, and consolidation."""
+
+from __future__ import annotations
+
+import hashlib
+import json
+import math
+import re
+import unicodedata
+from pathlib import Path
+from typing import Any, Iterable
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)